static/app/components/searchSyntax/renderer.tsx

```diff
@@ -144,7 +151,74 @@ function FilterToken({
         <KeyToken token={filter.key} negated={filter.negated} />
         {filter.operator && <Operator>{filter.operator}</Operator>}
         <Value>{renderToken(filter.value, cursor)}</Value>
-      </Filter>
+      </TokenGroup>
+    </Tooltip>
+  );
+}
+
+function FreeTextToken({
+  filter,
+  cursor,
+}: {
+  cursor: number;
+  filter: TokenResult<Token.FreeText>;
+}) {
+  const isActive = isWithinToken(filter, cursor);
+
+  // This state tracks if the cursor has left the filter token. We initialize it
+  // to !isActive in the case where the filter token is rendered without the
+  // cursor initially being in it.
+  const [hasLeft, setHasLeft] = useState(!isActive);
+
+  // Used to trigger the shake animation when the element becomes invalid
+  const filterElementRef = useRef<HTMLSpanElement>(null);
+
+  // Trigger the effect when isActive changes to updated whether the cursor has
+  // left the token.
+  useEffect(() => {
+    if (!isActive && !hasLeft) {
+      setHasLeft(true);
+    }
+  }, [hasLeft, isActive]);
+
+  const showInvalid = hasLeft && !!filter.invalid;
+  const showTooltip = showInvalid && isActive;
+
+  const reduceMotion = useReducedMotion();
+
+  // Trigger the shakeAnimation when showInvalid is set to true. We reset the
+  // animation by clearing the style, set it to running, and re-applying the
+  // animation
+  useEffect(() => {
+    if (!filterElementRef.current || !showInvalid || reduceMotion) {
+      return;
+    }
+
+    const style = filterElementRef.current.style;
+
+    style.animation = 'none';
+    void filterElementRef.current.offsetTop;
+
+    window.requestAnimationFrame(
+      () => (style.animation = `${shakeAnimation.name} 300ms`)
+    );
+  }, [reduceMotion, showInvalid]);
+
+  if (!filter.invalid?.reason) {
+    return <Fragment>{renderToken(filter.value, cursor)}</Fragment>;
```

Comment:
I think if it's not invalid we would just want this right?
```suggestion
    return <Fragment>{filter.text}</Fragment>;
```

This is what's causing the issue with quotes disappearing

You won't need this line then either https://github.com/getsentry/sentry/pull/49604/files#diff-26191720cdda981a3d270fcd73aa608c483a4b08e9ba758bb0adc244c12fce43R40

Reply:
oh yes you are right...thank you